cmd/crc/cmd/config/config.go

```diff
@@ -52,22 +52,17 @@ func RegisterSettings(cfg *config.Config) {
 }
 
 func isPreflightKey(key string) bool {
-	return strings.HasPrefix(key, "skip-") || strings.HasPrefix(key, "warn-")
+	return strings.HasPrefix(key, "skip-")
 }
 
 // less is used to sort the config keys. We want to sort first the regular keys, and
-// then the keys related to preflight starting with a skip- or warn- prefix. We want
-// these preflight keys to be grouped by pair: 'skip-bar', 'warn-bar', 'skip-foo', 'warn-foo'
+// then the keys related to preflight starting with a skip- prefix. We want
+// these preflight keys to be grouped as: 'skip-bar', 'skip-foo'
```

Comment:
The last sentence is no longer useful ("We want these preflight keys to be grouped ..."), it can be removed.